docs/specification/current.md

```diff
@@ -555,16 +546,11 @@ The following example illustrates a delete action:
 }
 ```
 
-#### OverrideReason
-
-An **OverrideReason** is described by the following attributes. This specification does not prescribe a standard set of override reasons; implementers are encouraged to submit suggestions for standardization. 
+#### overrideReasons
```

Comment:
Suggest removing this addition. Throughout the rest of the spec we document each data element as if it were a type. For example, `links` is an array of *Link*. The documentation describes *Link*, not `links`. If we want to add this overrideReasons header, we'd need to also add a links, suggestions and cards header.

Reply:
I re-titled this to **Reasons for rejecting a card** since the discussion in that paragraph needs a title separating it from what is above.

Note that there is a separate **OverridReason** type under Feedback that is still necessary.